SwiftMessageBar/SwiftMessageBar.swift

```diff
@@ -185,6 +203,22 @@ public final class SwiftMessageBar {
     tapHandler?()
   }
   
+  private func forceDismissMessage(_ message: Message) {
+    self.isMessageVisible = false
+    message.removeFromSuperview()
+    
+    if !self.messageQueue.isEmpty {
+        guard let message = messageQueue.dequeue() else {
```

Comment:
The method name is a bit misleading. It's called once to dismiss a message but then it also loops through the entire messageQueue to dismiss all queued up messages which comes as an unexpected side effect.
I don't quite get why it does this or why it has to dequeue and then remove the messages one by one. Wouldn't just clearing the messageQueue in one go have the same effect?